content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -362,7 +363,7 @@ status condition based on the state you determine:
 For `Approved` CSRs:
 
 ```yaml
-apiVersion: certificates.k8s.io/v1
+apiVersion: certificates.k8s.io/v1beta1
```

Comment:
I'd suggest we use the `v1` API because that is the GA'ed version.

Reply:
makes sense.. fixed in next commit.